front-end/src/screens/CreatePost/TopicsRadio.tsx

```diff
@@ -33,8 +33,7 @@ const TopicsRadio = ({ className, onTopicSelection }: Props) => {
 	}
 
 	return (
-		<Form.Field className={className}>
-			<label>Topic</label>
+		<Form.Field className={className} title='topic'>
```

Comment:
What does this `title` do?

Reply:
Should work as an accessibility alternative after having deleted the label

Reply:
hmm I can't see any such prop on Form or Form.Field though https://react.semantic-ui.com/collections/form/

Reply:
Yeah, it's a global field on any html element. But from the search I did on it, they are actually also not the best accessibility thing to have. So I'll remove it. 

Visually I still would like to get rid off the topics label; having radio buttons with categories is a pretty obvious sign for a user that these are tags.